package.json

```diff
@@ -5,32 +5,33 @@
   "description": "Kuma Manager",
   "author": "Kong",
   "scripts": {
-    "serve": "VUE_APP_MOCK_API_ENABLED=true yarn vue-cli-service serve",
-    "serve:with-api": "yarn vue-cli-service serve",
-    "build": "yarn vue-cli-service build",
-    "build:ci": "VUE_APP_MOCK_API_ENABLED=true yarn run build --mode development",
-    "lint": "yarn vue-cli-service lint",
-    "test:unit": "yarn vue-cli-service test:unit",
-    "test:unit:ci": "yarn run test:unit --runInBand",
+    "dev": "VITE_MOCK_API_ENABLED=true vite",
+    "dev:with-api": "vite",
+    "preview": "vite preview",
+    "build": "vite build",
+    "build:ci": "VITE_MOCK_API_ENABLED=true yarn run build --mode development",
+    "lint": "eslint --ext .js,.ts,.vue --fix .",
+    "test:unit": "jest",
     "test:unit:watch": "yarn run test:unit --watch"
   },
   "dependencies": {
     "@amcharts/amcharts4": "^4.10.27",
     "@appscode/json2yaml": "^0.1.2",
-    "@datadog/browser-logs": "^4.17.2",
+    "@datadog/browser-logs": "^4.19.0",
```

Comment:
I know it's not new but should we be collecting logs from the GUI?

Reply:
FYI @lahabana ☝️

Reply:
I’m not sure we should be collecting logs for Datadog specifically, but I would like to consider _some_ form of error observability like Sentry which in my opinion is really valuable.

Reply:
Turns out we don’t initialize Datadog by default: https://github.com/kumahq/kuma-gui/blob/master/src/datadog.ts#L6-L8

Reply:
then great, thanks! 👍